test/langtools/tools/javac/launcher/src/p/q/CLTest.java

```diff
@@ -1,10 +1,12 @@
 /*
- * Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Probably:
```suggestion
 * Copyright (c) 2018, 2023, Oracle and/or its affiliates. All rights reserved.
```
(not sure, but the file existed before, it is just moved, AFAIK.)

Reply:
Yes. It was moved from `.../launcher/src` to `.../launcher/p/q`, thus committing as suggested.